### PR TITLE
Add 2000bp size check for raw bed class

### PR DESCRIPTION
**Pull request by Jmeigs1**
Notes:

1.  Should a similar check be added for the covered class?  Based on my understanding the covered class represents the output where the raw class is the input so I imagine the constraints might be different.

2.  The end result of these "fatal" errors is that the line is excluded from the bed file that is eventually passed to the c binary and execution continues.  Is this reasonable?  Based on the name alone it seems like execution should stop and report an error to the user.  Does it make sense high level to return a partial solution for the bed lines devoid of clerical errors?

### CODE REVIEW

File: lib/MPD/Bed/Raw.pm [hunk 1]
```diff
@@ -60,6 +60,11 @@ sub BUILD {
       $self->Chr, $self->Start, $self->End );
     return $self->log( 'fatal', $msg );
   }
+  if ( $self->Size > 2000 ) {
```
Comment (akotlar) [inline lib/MPD/Bed/Raw.pm:63]: Want to make sure this is something Thomas or Dave approved. Not sure what the right number here is

**Review by akotlar — CHANGES_REQUESTED** [review]
Double check 2000bp

File: lib/MPD/Bed/Raw.pm (hunk 1 above)
Comment (Jmeigs1) [reply lib/MPD/Bed/Raw.pm:63]: 2000 was Thomas' suggestion.  Should have made that clear in OP.

**Review by akotlar — APPROVED** [review]

File: lib/MPD/Bed/Raw.pm (hunk 1 above)
Comment (wingolab) [reply lib/MPD/Bed/Raw.pm:63]: This is an arbitrary decision that is based on my experience. We could empirically decide the right answer using testing but I am not sure that would be worth while at this time.

**Review by wingolab — APPROVED** [review]

**Comment by Jmeigs1** [comment]
@wingolab Please don't close.  The code is not merged into the main repo and there are unaddressed concerns from my original post.

**Comment by wingolab** [comment]
Thank you. That was an accident.

**Comment by Jmeigs1** [comment]
Don't need check for covered class.  Confirmed fatal error should prevent further computation.  Marked the try catch capturing the error for future review.

**Pull request merged**